ReactiveObjC/RACSignal.h

```diff
@@ -472,6 +489,18 @@ typedef RACSignal * _Nullable (^RACSignalBindBlock)(ValueType _Nullable value, B
 ///
 /// Returns whether the signal completed successfully before timing out. If NO,
 /// `error` will be set to any error that occurred.
+- (BOOL)asynchronouslyWaitUntilCompleted:(NSError * _Nullable * _Nullable)error timeout:(NSTimeInterval)timeout;
+
+/// Spins the main run loop for a short while, waiting for the receiver to complete
+/// or the provided timeout to elapse.
```

Comment:
It appears that this doc comment is for the wrong method—this method `asynchronouslyWaitUntilCompleted:` does not take "a provided timeout"—rather the one above this `asynchronouslyWaitUntilCompleted:timeout:` does.